salt/spm/__init__.py

```diff
@@ -359,6 +359,7 @@ def _install(self, args):
 
         # First we download everything, then we install
         for package in dl_list:
```

Comment:
I believe @techhat did this so that the bzip2 tarballs would be all downloaded first before trying to install them like yum/apt/pacman and other package managers do.